std-bits/base/src/main/java/org/enso/base/DryRunFileManager.java

```diff
@@ -0,0 +1,40 @@
+package org.enso.base;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DryRunFileManager {
+  static final Map<String, String> files = new HashMap<>();
+
+  /**
+   * Creates a temporary file for the given name.
```

Comment:
I'd like the comment to explain the behaviour a bit more, if possible:
- if `name` is a path to a file that is a temporary file associated with some other file, it is also returned - but no new temporary file is returned - its "temporary file" is "itself".

(That's not good phrasing for the comment, but I think it would be good to express that idea in the comment - otherwise it is a bit surprising to see this behaviour, it's not obvious that this is what we want (even though I know it is))